packages/gatsby-graphiql-explorer/index.js

```diff
@@ -0,0 +1,19 @@
+const path = require(`path`)
+
+module.exports = (expressApp, { graphqlEndpoint, getFragments }) => {
+  const bundleUrlHandler = path.posix.join(graphqlEndpoint, `app.js`)
+  const fragmentsUrlHandler = path.posix.join(graphqlEndpoint, `fragments`)
+
+  expressApp.get(bundleUrlHandler, (_req, res) => {
+    res.set(`Cache-Control`, `public, max-age=31557600`)
```

Comment:
this `Cache-Control` (which I know is copied from current stuff), in general seems problematic because it does force "force refresh" or disabling browser cache. So not for this PR, but probably something for next time we mess with with graphiql app (or maybe add `etag` somehow to be able to invalidate browser cache)